src/Api/FingerprintApi.php

```diff
@@ -134,11 +137,16 @@ public function getEventWithHttpInfo($request_id)
 
             $responseBody = $response->getBody()->getContents();
 
-            $serialized = ObjectSerializer::deserialize($responseBody, $returnType, []);
-            $serialized->setRawResponse($responseBody);
+            $serialized = null;
+            try {
+                $serialized = ObjectSerializer::deserialize($responseBody, $returnType, []);
+            } catch (\Exception $e) {
+                return [null, $responseBody, $response->getStatusCode(), $response->getHeaders()];
```

Comment:
Shouldn't we propagate the exception somehow to the customer? Otherwise he won't be able to find out what went wrong